src/components/tooltip/README.md

```diff
@@ -38,11 +38,23 @@ This tooltip variation does not use any JavaScript and should be used to define
 | .bx--tooltip--definition__bottom | A simple tooltip that is displayed below the trigger. |
 
 
-### Simple tooltip
+### Icon tooltip
 
 This tooltip variation does not use any JavaScript and is good for short single line of text describing an icon. No label should be added to this variation.
 
 | Selector                     | Description                                           |
 |------------------------------|-------------------------------------------------------|
 | .bx--tooltip--icon__top      | A simple tooltip that is displayed above the trigger. |
 | .bx--tooltip--icon__bottom   | A simple tooltip that is displayed below the trigger. |
+
+## Accessibility Notes
```

Comment:
Need to clean up the notes section here, as this is just what I jotted down quickly.

Reply:
Do we need this section in the readme now? Should we create an issue to spend more time on writing out guidance for this instead in the future?

Reply:
Can definitely remove it, that's fine 👍